### ISSUE TITLE
Update to Bell Museum request for new collections (Issue #8227)

### ISSUE THREAD

**Issue by adhornsby** [ArctosDB/arctos#8387]
Apologies to open a new issue for this @mkoo , the original got assigned to a repo I don't have access to so I didn't have a way to follow up.

**ORIGINAL ISSUE**: [ArctosDB/arctos] Request - new collections MMNH:Egg and MMNH:Para (Issue #8227)
Bell Museum wants to add MMNH:Egg and MMNH:Para.

**UPDATE TO ISSUE** (this post)
In addition to MMNH:Egg and MMNH:Para, we'd also like to add JFBM:Amph and JFBM:Rept, and we'd like to DELETE JFBM:Herp [empty].  (Reason being: our Amph and Rept collections used to have separate catalogs, then were forced together as an alphanumeric series some years ago.  We're planning to re-separate them so we can use strictly numeric catalog numbers again and avoid renumbering anything.  Let me know if you see any red flags in that plan.)

It sounds from Dusty like these changes need to be reflected in our MOU before proceeding (https://github.com/ArctosDB/internal/issues/354).  If this doesn't need to wait on the MOU, I'm happy to get the process started by submitting the requests according to https://arctos.database.museum/Admin/pre_collection.cfm
